CameraOverlay/overlay_all_stats.py

```diff
@@ -10,13 +10,15 @@ class OverlayAllStats(Overlay):
 			topics.DAS.stop,
 			topics.PowerModel.recommended_sp,
 			topics.PowerModel.max_speed,
-			topics.DAShboard.recieve_message,
+			topics.DAShboard.receive_message,
 	]
 
 	def __init__(self):
 		super(OverlayAllStats, self).__init__()
 		self.text_height = 50
 		self.speed_height = 70
+		self.message_received_time = 0
+		self.message_received_flag = False
```

Comment:
`self.displaying_message` might be more descriptive, as it tells the reader that this will go back to `False` when we hide the message. (To me, `message_received_flag` might mean something like a message has been received and needs processing)